gradle.properties

```diff
@@ -17,12 +17,12 @@ junitVersion=4.12
 lombokVersion=1.18.4
 mockitoVersion=1.10.19
 scalaVersion=2.11.8
-shadowGradlePlugin=2.0.3
+shadowGradlePlugin=6.1.0
```

Comment:
Is this necessary to upgrade to the next Pravega snapshot or is this version change simply piggybacking on to this PR?

Reply:
This PR is to fix the build, which comes before doing any release-related activities.